package.json

```diff
@@ -51,6 +51,7 @@
     "jest-resolve": "26.6.0",
     "jest-watch-typeahead": "0.6.1",
     "lodash": "^4.17.21",
+    "mime": "^1.6.0",
```

Comment:
Chyba ta biblioteka nie jest nigdzie używana?

![image](https://user-images.githubusercontent.com/21079319/116479056-5973b400-a87f-11eb-9d65-1ade3850ce67.png)

Reply:
Dzięki, mój błąd!

Reply:
Jak coś, to usunąłem, bo były jakieś problemy z package-lockiem